### PR TITLE
[CMGR-50790] update tutorial to use OAuth instead of JWT

### PR DESCRIPTION
**Pull request by amisha1609**
## Description

[CMGR-50790](https://jira.corp.adobe.com/browse/CMGR-50790)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/tutorial/4-getting-an-access-token.md [hunk 1]
```diff
@@ -16,46 +16,38 @@ import Glitch from "../../components/glitch"
...
-    orgId: process.env.ORGANIZATION_ID,
-    clientSecret: process.env.CLIENT_SECRET,
-    metaScopes: [ 'ent_cloudmgr_sdk' ]
+  const form = new FormData();
+  form.append('client_id', process.env.CLIENT_ID);
+  form.append('client_secret', process.env.CLIENT_SECRET);
+  form.append('grant_type', process.env.GRANT_TYPE);
+  form.append('scope', process.env.SCOPES);
+
+  const response = await fetch('https://ims-na1.adobelogin.com/ims/token/v3', {
```
Comment (kritikash18) [inline src/pages/tutorial/4-getting-an-access-token.md:41]: I wouldn't recommend doing it this way. Earlier also there were 2 API calls but they were abstract under @adobe/jwt-auth library. I believe there would be a similar adobe library for oauth, please use that

**Review by kritikash18 — CHANGES_REQUESTED** [review]

File: src/pages/tutorial/4-getting-an-access-token.md (hunk 1 above)
Comment (amisha1609) [reply src/pages/tutorial/4-getting-an-access-token.md:41]: I cannot find any such library for oauth. can you please share if you know it?

Comment (kritikash18) [reply src/pages/tutorial/4-getting-an-access-token.md:41]: [https://github.com/adobe/aio-lib-ims-oauth](https://github.com/adobe/aio-lib-ims-oauth)

File: src/pages/tutorial/4-getting-an-access-token.md [hunk 2]
```diff
@@ -16,46 +16,38 @@ import Glitch from "../../components/glitch"
...
+  })
+  if (!response.ok) {
+    throw new Error('Failed to get access token');
   }
-  config.privateKey = fs.readFileSync('.data/private.key')
-
-  const { access_token } = await auth(config)
-  return access_token  
+  const responseData = await response.json();
+  return responseData.access_token;
```
Comment (kritikash18) [inline src/pages/tutorial/4-getting-an-access-token.md:50]: please test this piece of code thoroughly

**Review by kritikash18 — APPROVED** [review]

File: src/pages/tutorial/4-getting-an-access-token.md (hunk 2 above)
Comment (kritikash18) [reply src/pages/tutorial/4-getting-an-access-token.md:50]: also fix failing checks

**Pull request merged**